listenbrainz/webserver/static/js/src/BrainzPlayer.tsx

```diff
@@ -479,6 +486,59 @@ export default class BrainzPlayer extends React.Component<
     trailing: true,
   });
 
+  submitListenToListenBrainz = async (
+    title: string,
+    trackURL: string,
+    artist?: string,
```

Comment:
`artist_name` is always needed. can remove the `?`

Reply:
Ah ok! I had thought this came from the current listen instead of what's playing in BP so we'd have an artist always.